custom_image/worker/README.md

```diff
@@ -0,0 +1,135 @@
+
+
+7. Create a directory (/tmp/packages/symphony) and copy the listed packages.
+
+   ```cli
+   $ mkdir -p /tmp/packages/symphony
+   $ unzip sym-7.3.2.0_x86_64_build601349.tar.gz # This unzips the required Symphony build601349 fix patches
+   ```
+   
+   Example:
+   ```cli
+   $ ls /tmp/packages/symphony
+   sym-7.3.2.0_x86_64.bin                                   
+   egocore-4.0.0.0_x86_64_build601349.tar.gz       
+   egoelastic-1.4.5.0_x86_64_build601349.tar.gz    
+   egomgmt-4.0.0.0_noarch_build601349.tar.gz       
+   egorest-4.0.0.0_noarch_build601349.tar.gz       
+   explorer-10.1.1.0_x86_64_build601349.tar.gz     
+   hfcore-1.2.0.0_x86_64_build601349.tar.gz
+   hfmgmt-1.2.0.0_noarch_build601349.tar.gz
+   soamcore-7.3.2.0_x86_64_build601349.tar.gz
+   soammgmt-7.3.2.0_noarch_build601349.tar.gz
+   soammrcore-7.3.2.0_x86_64_build601349.tar.gz# optional 
+    ```
+   
+8. Download a pre-built [Packer binary](https://www.packer.io/downloads) for your operating system.
+
+## Create Custom Image (using Packer)
+
+The following steps provision the IBM Cloud VSI, install the following Spectrum components, and create a new image.
+   - Spectrum Scale
+   - Symphony
+   - Symphony Patches
+
+   **Note**: The [script.sh](script.sh) provides stepwise installation flow for all of the above components. By commenting the component installation stanza, you can skip the unnecessary component installation and reduce the image size.
+
+1. Change working directory to `custom_image/worker/`.
+
+   ```cli
+   cd hpc-cluster-symphony/custom_image/worker/
+   ```
+
+2. Create a Packer variable definitions file (`inputs.auto.pkrvars.hcl`) and provide infrastructure inputs.
+    
+   **Note**: The `vpc_subnet_id` and `source_image_name` should belong to `vpc_region` variable value. 
+
+   Minimal Example:
+
+   ```jsonc
+   $ cat inputs.auto.pkrvars.hcl
+   ibm_api_key = "<IBMCloud_api_key>"
+   vpc_region = "<IBMCloud_supported_region_name>"
+   resource_group_id = "<Existing_resource_group_id>"
+   vpc_subnet_id = "<Existing_subnet_id_from_provided_vpc_region>"
+   source_image_name = "<Source_image_name_from_images_of_VPC>"
+   image_name = "<image_name_for_newly_created_custom_image>"
+    ```
+
+3. Run `packer init .` to install the Packer plugin for IBM Cloud.
+
+
+4. Run `packer build .` to create the custom image.
+
+   **Note**: After a successful Packer build, you should be able to see a newly created custom image at [Images of VPC](https://cloud.ibm.com/vpc-ext/compute/images) with the name you provided for `image_name` parameter with Active status.
+
+
+## Limitations:
+
+- The custom image creation scripts are only compatible with Red Hat-based Linux distributions operating systems.
+- The custom image creation scripts support only VSI based deployment models for both Symphony (where worker_type is specified as VSI) and Scale (where storage_type is specified as scratch). 
```

Comment:
Do we have a Github issue open (and in backlog) to add custom image creation script support for deployment models that make use of bare metals (for worker nodes, and for Scale persistent storage nodes)?